enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import type { StoryFn } from "@storybook/react";
+
+import { InteractiveQuestion } from "embedding-sdk";
+import type { InteractiveQuestionProps } from "embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion";
+import { CommonSdkStoryWrapper } from "embedding-sdk/test/CommonSdkStoryWrapper";
+
+const QUESTION_ID = (window as any).QUESTION_ID || 1;
+
+export default {
+  title: "EmbeddingSDK/InteractiveQuestion",
```

Comment:
TODO: write Cypress e2e tests for InteractiveQuestion once I got the E2E set up locally. I tried to write a unit test with react-testing-library, but the click actions popover would not trigger.